soc/nordic/common/vpr/Kconfig.sysbuild

```diff
@@ -4,7 +4,7 @@
 config VPR_LAUNCHER
 	bool "VPR launcher"
 	default y
-	depends on (SOC_NRF54H20_CPUPPR || SOC_NRF54H20_CPUFLPR || SOC_NRF54L15_CPUFLPR || SOC_NRF9280_CPUPPR)
+	depends on (SOC_NRF54H20_CPUPPR ||  SOC_NRF54H20_CPUFLPR ||  SOC_NRF54L_05_10_15_CPUFLPR || SOC_NRF9280_CPUPPR)
```

Comment:
This change should be reverted, since an option like `SOC_NRF54L_05_10_15_CPUFLPR` does not exist. But since FPLRs are not supported for L05 and L10 boards, nothing needs to be changed here.